taskcluster/tf_tc-build.sh

```diff
@@ -4,56 +4,48 @@ set -ex
 
 source $(dirname $0)/tf_tc-vars.sh
 
-build_amd64=yes
+build_amd64=no
 build_gpu=no
 build_android_arm=no
 build_android_arm64=no
 build_linux_arm=no
 build_linux_arm64=no
+build_ios_arm64=no
```

Comment:
Given those changes, I think it might be beneficial now to move to `case`-based definition instead of the simpler `if` approach we had here, which is already mostly done: it would be easier to read and ensure we are straight

Reply:
I don't know enough bash to understand what you want me to do.

Reply:
Simply change the bunch of `if` that sets the variables + `if` that uses the variable to a `case`-based switch that calls the build

Reply:
What about the cases that share steps, like `--gpu`, `--arm` and `--arm64`? Switching to a case that calls the build directly would require us to duplicate the build calls there, right?

Reply:
Ah, nevermind, I see that nothing is actually being shared.